docs/best-practices.md

```diff
@@ -290,7 +290,59 @@ sumologic:
 
 ## Collect logs from additional files on the Node
 
-:construction: _TODO_, see [the Fluentd section](fluent/best-practices.md#add-a-local-file-to-fluent-bit-configuration)
+To collect logs from additional files on Node, it is necessary to:
+
+- mount log file to make it accessible by log collector pod
+- create an additional pipeline in both the log collector and metadata enrichment service to transfer your data
+- open a new port for this pipeline to enable sending data from the log collector to the metadata enrichment service
+
+The following configuration can be used for these purposes:
+
+```yaml
+otellogs:
+  config:
+    merge:
+      receivers:
+        filelog/extrafiles:
+          include: [/var/log/extrafiles/extrafile.log]
```

Comment:
Can you also provide an example for receiving data from the SystemD Journal?  In particular, this approach looks very close to supporting collection of IPTables "LOG" directives.

Reply:
systemd logs are already collected by default, you don't need any additional configuration for those.